### ISSUE TITLE
Build Errors

### ISSUE THREAD

**Issue by ValliteKing** [Luukus/SWBF2-Classic-Hack#1]
Attempting to build spits out numerous errors. Tested on VS2019 and VS2022.

Errors below.

![Screenshot 2023-10-11 164908](https://github.com/Luukus/SWBF2-Classic-Hack/assets/50279426/faeed101-a161-4876-a890-36e5a7a7e69e)

**Comment by TyeStanley** [comment]
I'm also getting those same build errors... I thought it was something on my end! Hopefully, he could get this fixed because I was really looking forward to trying it out!

**Comment by Luukus** [comment]
It looks like it's not referencing the path to the externals correctly. Is the `externals` directory in the root of the project still? If so, make sure the path to the externals directory is configured correctly as shown in this image

![externals](https://github.com/Luukus/SWBF2-Classic-Hack/assets/142463408/6c44a0dc-b060-40da-b892-3e48eb235522)

Make sure this is configured for both Debug and Release

**Comment by ValliteKing** [comment]
Nope, still getting the same errors. Applied for both release and debug. Tried on both VS2019 and VS2022.

**Comment by Luukus** [comment]
> Nope, still getting the same errors. Applied for both release and debug. Tried on both VS2019 and VS2022.

Can you share a screenshot of what your "Include Directories" are configured for? You need to make sure it's configured for all platforms and configurations.
